packages/editor-ui/src/stores/nodeCreator.ts

```diff
@@ -166,9 +166,11 @@ function recommendedCategory(
 				i18n.baseText('nodeCreator.actionsCategory.recommended'),
 			),
 			actionKey: categoryItem.value as string,
-			displayName: i18n.baseText('nodeCreator.actionsCategory.onEvent', {
-				interpolate: { event: startCase(categoryItem.name) },
-			}),
+			displayName:
+				categoryItem.action ??
```

Comment:
Are we okay with no i18n in this case?

Reply:
This override comes from the Node definition for which we're not using i18n

Reply:
Exactly! In almost all cases this string is translatable but for overrides it's not, so not ideal for someone localizing. Just to confirm this is okay, minor thing.